cmd/loopd/main.go

```diff
@@ -11,5 +12,6 @@ func main() {
 	err := loopd.Run(cfg)
 	if err != nil {
 		fmt.Printf("loopd exited with an error: %v\n", err)
+		os.Exit(1)
```

Comment:
One question is why the print statement above doesn't show up in the issue log. Maybe diff version?

Reply:
Yes, the error was added in a PR that didn't make it into a release yet.